### DISCUSSION TITLE
How to predict the charge via DeepMDkit

### DISCUSSION THREAD

**Discussion by zhengcheng233** [deepmodeling/deepmd-kit#886] [Q&A]
Recently, I want to calculate the charges of some small polypeptides. I tried to collect all the charges of structures in training set as atom_ener.npy, and the dp training process seem to be succeeded. 
After training, I freeze the dp model as graph.pb and using dp = DP('graph.pb') to load the parameters. However, when i use "dp.eval(coord,cell,atype,atomic=True", the program can not give the atomic contribution. So is there any approach to obtain the atomic contribution of the Neural networks.

**Comment by njzjz** [comment]
Which version are you using?
